pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -1392,131 +1609,1374 @@ def marker_minimum(self):
+        to set the reference level.
         """
-        Compensate for losses outside the instrument when in external
-        mixer mode (such as losses within connector cables, external mixers, etc.).
-        'conversion_loss' specifies the mean conversion loss for the current harmonic band.
-        In a full frequency band (such as band K), the mean conversion loss is defined as the
-        minimum loss plus the maximum loss for that band divided by two.
-        Adjusting for conversion loss allows the system to remain
-        calibrated (that is, the displayed amplitude values have the conversion loss incorporated
-        into them). The default value for any band is 30 dB. The spectrum analyzer must be in
-        external-mixer mode in order for this command to work. When in internal-mixer mode,
-        querying 'conversion_loss' returns a zero.
-        """,
-        validator=strict_range,
-        values=[0, float("inf")]
-    )
+        self.write("MKRL")
 
-    def fullband(self, band):
+    def marker_delta_to_span(self):
+        """Set the frequency span equal to the frequency difference between two
+        markers on a trace.
+
+        The start frequency is set equal to the frequency of the left-
+        most marker and the stop frequency is set equal to the frequency
+        of the right-most marker.
         """
-        Select a commonly-used, external-mixer frequency band, as shown
-        in the table. The harmonic lock function :attr:`harmonic_number_lock` is also set; this
-        locks the harmonic of the chosen band. External-mixing functions are not available with
-        an HP 8560A Option 002. Takes frequency band letter as string.
+        self.write("MKSP")
 
-        .. list-table:: Title
-            :widths: 25 25 25 25
-            :header-rows: 1
+    def marker_to_center_frequency_step_size(self):
+        """Set the center frequency step-size equal to the frequency value of
+        the active marker."""
+        self.write("MKSS")
 
-            * - Frequency Band
-              - Frequency Range (GHz)
-              - Mixing Harmonic
-              - Conversion Loss
-            * - K
-              - 18.0 — 26.5
-              - 6
-              - 30 dB
-            * - A
-              - 26.5 — 40.0
-              - 8
-              - 30 dB
+    marker_time = Instrument.control(
+        "MKT?", "MKT %gS",
+        """
+        Place a marker at a position that corresponds to a specified point in
+        time during the sweep. Default units are seconds.
+
+        Type: :code:`float`
+
+        .. code-block:: python
+
+            # set marker at sweep time corresponding second two
+            instr.marker_time = 2
+
+            if instr.marker_time == 2:
+                pass
+
+        """
+    )
+
+    marker_signal_tracking = Instrument.control(
+        "MKTRACK?", "MKTRACK %s",
+        """
+        Locate the active marker and sets the center frequency to the
```

Comment:
Please start the docstring with `Control` to indicate the type of property (the API does not show, whether it is a control or measurement or setting).